pointcloud_roi:
  ros__parameters:
    input_topic: "/velodyne_points"
    output_topic: "/roi_cloud"
    x_min: 0.0
    x_max: 5.0
    y_min: -5.0
    y_max: 5.0
    z_min: 0.0
    z_max: 2.0
